auth_plugins/rollcontrol/plugin.py

```diff
@@ -14,6 +16,11 @@
 class RollPlugin(BotAuthPlugin):
     def __init__(self, load_from_disk=True):
         super().__init__()
+        if os.getenv("PROMETHEUS_PORT"):
+            # start serving prometheus over port
+            start_http_server(int(os.getenv("PROMETHEUS_PORT")))
```

Comment:
```
start_http_server(int(os.getenv("PROMETHEUS_PORT")))
```

Should be

```
start_http_server(int(os.environ["PROMETHEUS_PORT"]))
```

Since we know the environment variable exists from your `if` logic you should just directly retrieve it as this is quicker [1]

[1] https://partofthething.com/thoughts/speed-of-python-dict-access-methods-direct-vs-get/